content/learn/book/welcome/_index.md

```diff
@@ -6,4 +6,44 @@ template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: Briefly discuss what Bevy is.
+Welcome to the Bevy game engine! A bevy may be a group of birds, but our engine is:
+
+- Rust-first: with all the safety, tooling and power that comes with a modern systems language
+- completely free, open source and hackable with an architecture designed for modularity
+- powered by an ergonomic and innovative ECS that makes writing clear, performant gameplay and engine logic a joy
+- cross-platform: support all major desktop platforms with a single code-base, with limited mobile and web support
+- designed for effortless parallelism with a data-oriented design and automatic parallel scheduler
+- built on a high-performance foundation
+
+This book is designed for new and experienced users looking for a thoughtful, [explanation-first](https://diataxis.fr/explanation/) guide to the engine's essential features.
+If you just want to try it out hands-on, jump into one of our [quick-start guides](TODO: add link) instead!
+
+## Stability warning
+
+Bevy aims to be a general-purpose game engine capable of handling arbitrary 2D or 3D games.
+However, Bevy is still in its infancy, and there are very good reasons not to choose it for your next project.
+It is currently completely missing first-party solutions for:
+
+- animation
+- networking
+- console support
+
+Solid third-party (but not first-party) solutions currently exist for:
+
+- [audio](https://crates.io/crates/bevy_kira_audio)
```

Comment:
People will likely read this and think Bevy has no support for audio at all. I don't like those optics.